lib/policyEvaluator/utils/objectTags.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Removes tag key value from condition key and adds it to value if needed
+ * @param {string} key - condition key
+ * @param {string} value - condition value
+ * @return {array} key/value pair to use
+ */
+function transformTagKeyValue(key, value) {
+    if (!key.includes('/')) {
```

Comment:
It might be a good idea not to include any pattern that we do not explicitly recognize and support and keep the default behavior for those. To do this, we could check explicitly each recognized prefix pattern `s3:ExistingObjectTag/` etc. instead of assuming that because there is a `/` it is a tag pattern, which looks somewhat hazardous to me.

Reply:
I was trying to keep the function generic so that it could be used in the future if we support other condition keys that use `/`, and there's no condition key that contains a `/` that wouldn't require transformation, but I can change it if you think it might introduce problems.

Reply:
just for the record, I added the test suggested below, and it does get transformed through this function, but because of the way condition keys are evaluated, it doesn't give permission. I personally don't think there's any harm in that sort of error handling, but will defer to your wisdom :p

Reply:
If the transformation that the function is doing makes sense for all cases where `key` contains a `/`, it's ok for me.